sqldb/sqlerrors.go

```diff
@@ -21,6 +21,7 @@ var (
 	postgresErrMsgs = []string{
 		"could not serialize access",
 		"current transaction is aborted",
+		"not enough elements in RWConflictPool",
```

Comment:
IIUC, this gets returned when the instance [runs out of shared memory](https://www.postgresql.org/message-id/4E202D52020000250003F3A1@gw.wicourts.gov). I wager this is popping up mainly due to the constrained environment that the CI runners execute on.